.contrib/Parser/DATAS/02 - Outdoor Zones/02 Eastern Kingdoms/Isle of Quel'Danas/Vendors.lua

```diff
@@ -1317,7 +1317,7 @@ _.Zones =
 					},
 				}),
 				n(25977, {	-- Yrma <Transmuter of Sin'dorei Relics>
-					["coord"] = { 50.2, 28.2 },
+					["coord"] = { 50.2, 28.2, 1270 },
```

Comment:
How was this determined? Asking because 1270 is a "new" map added in BfA and not 122 which is used in other places ("original" one).

Reply:
I used https://wowpedia.fandom.com/wiki/UiMapID.  I now do see two IDs for that zone 1270 and 122.  It believe that it should be 122 also.

Reply:
okay, I will fix it shortly